content/guides/docker-concepts/the-basics/what-is-a-container.md

```diff
@@ -115,7 +115,13 @@ The `docker/welcome-to-docker` container continues to run until you stop it.
 
 Follow the instructions to run a container using the CLI:
 
-1. Open your CLI terminal and start a container by using the [`docker run`](/reference/cli/docker/container/run/) command:
+1. Open your CLI terminal and pull the welcome-to-docker image from Dockerhub by using the [`docker pull`](/reference/cli/docker/compose/pull/) command:
+
+    ```console
+    $ docker pull docker/welcome-to-docker
+    ```
+
+2. Start a container by using the [`docker run`](/reference/cli/docker/container/run/) command:
```

Comment:
`docker run` will automatically pull the image if it doesn't already exist in your local image store. This step is not necessary so I suggest we revert this bit.

```suggestion
1. Open your CLI terminal and start a container by using the [`docker run`](/reference/cli/docker/container/run/) command:
```

Reply:
Thanks. I was not aware about it. 

How about we add this statement in that point itself just to clarify that docker pulls the image if it is not present ??

Reply:
Yeah, maybe. I think we can leave it out for now but if it seems to cause more confusion I'll revisit.